src/btree/bt_delete.c

```diff
@@ -195,14 +195,14 @@ __wt_delete_page(WT_SESSION_IMPL *session, WT_REF *ref, bool *skipp)
     *skipp = true;
     WT_STAT_CONN_DATA_INCR(session, rec_page_delete_fast);
 
-    /* Publish the page to its new state, ensuring visibility. */
+    /* Release the page to its new state, ensuring visibility. */
```

Comment:
May be just say /* set the page to its new state */ I think ensuring visibility is also misleading.

Reply:
SGTM. We should also do the same with `Release the page to its previous state, ensuring visibility.` a few lines below and make it "restore" instead of "release"

Reply:
b06cc6cec